neuralmonkey/nn/bidirectional_rnn_layer.py

```diff
@@ -43,34 +43,33 @@ def outputs_bidi(self):
 
     @property
     def encoded(self):
-        """States of the bidirectional layer"""
+        """Last state of the bidirectional layer"""
         return tf.concat(1, [self._last_state, self._last_state_rev])
 
 
-    def _reverse_seq(self, input_seq, lengths):
-        """Reverse a list of Tensors up to specified lengths.
+def _reverse_seq(input_seq, lengths):
```

Comment:
Ta funkce je už v TF 0.9. A asi by to chtělo napsat do README, že používáme TF 0.9 :-)